src/commands/run-tests-solidus-older.yml

```diff
@@ -3,6 +3,8 @@ description: >
 
 steps:
   - test-branch:
-      branch: v3.0
+      branch: v3.2
```

Comment:
This should be already covered by `run-tests-solidus-current`

Reply:
Oh, good catch, thank you. Fixed.